### DISCUSSION TITLE
create organisation , doman, ip pool

### DISCUSSION THREAD

**Discussion by z3d3m0n** [postalserver/postal#1833] [Help with using Postal]
Hello Community,

glad to be a part 
hope everyone is safe in this not an easy time

I express my deepest gratitude to the author for  a great job
the system works smoothly straight out of the box.

i have a few questions regarding the API an Functionality.

1. is there any automated way how to create an organization , assign ip pool and add domain ? API , SDK?
2. is there hidden spot where i can set up a domain / MX policy ( eg  set up a msg rate for certain mx servers like proofpoint , or gmail  to avoid trouble ? )
3. is there a way to apply a back off rules/ policies for IP's?

any advises or comments are welcome
thx

**Comment by willpower232** [comment]
1. I think someone has suggested it in the past and there may be a PR somewhere for it but its not currently a feature of Postal
2. there is no domain-specific policy feature, all messages will be attempted unless the credential says otherwise
3. if a message fails to be delivered, it will be retried if possible with an increasing amount of time before each retry but this cannot be customised per recipient

**Reply by z3d3m0n** [reply]
@willpower232 

noted
thank you for your response
